scripts/provisioning/configcmd.py

```diff
@@ -104,7 +104,7 @@ def process(self, skip_openldap = False, skip_haproxy = False):
       self.create_auth_jks_password()
       self.logger.info('create auth jks password completed')
 
-      if skip_openldap == False:
+      if(self.services is None or 'openldap' in self.services):
         # Configure openldap only
         self.configure_s3_schema()
```

Comment:
if services is openldap, it should just push openldap schema and exit, it shouldn't continue with other steps

Reply:
I think this is correct @bkirunge7 We want to run 'full' provisioning in control pod as well - right ? (which will be overwritten by storage pod)

Reply:
If POD restarts, we should check first ldap s3 schema exists, then proceed. In this case we may to skip all other configurations too, so it will be better to separate out. 
Fine if existing method works

Reply:
Have put the condition -
if ldap schema exists then it will skip ldap schema config and proceed with rest everything